### DISCUSSION TITLE
Any plans to remove selenium-wire since deprecated?

### DISCUSSION THREAD

**Discussion by edudelta** [seleniumbase/SeleniumBase#2780] [Q&A] [answered]
Selenium-wire is deprecated since January 2024. 
Do you have any plans to remove or change to use another solution? tks

**Comment by mdmintz** [comment, marked as answer]
SeleniumBase's `wire` mode still works: https://github.com/seleniumbase/SeleniumBase/issues/2774#issuecomment-2110911078

First install the extra dependencies:

```bash
pip install seleniumbase[selenium-wire]
```

Then run a `wire` mode script:

```python
from seleniumbase import SB

with SB(wire=True) as sb:
    sb.driver.get("https://wikipedia.org")
    for request in sb.driver.requests:
        print(request.url)
```

There was a lot of documentation on it here: https://github.com/seleniumbase/SeleniumBase/issues/2145

----

SeleniumBase also has alternatives that are built in. See the examples:

* [SeleniumBase/examples/raw_cdp_logging.py](https://github.com/seleniumbase/SeleniumBase/blob/master/examples/raw_cdp_logging.py)
* [SeleniumBase/examples/uc_cdp_events.py](https://github.com/seleniumbase/SeleniumBase/blob/master/examples/uc_cdp_events.py)
